src/pages/journal/journal.html

```diff
@@ -14,6 +14,88 @@
     <ion-card-content>
       <button ion-button navPush="ContactDetailsPage">Contact Details</button>
       <button ion-button navPush="WaitingRoomPage">Waiting Room</button>
+
+      <div class="mes-text-bold-black-xxlarge"> .mes-text-bold-black-xxlarge </div>
```

Comment:
Should we use h1 instead?

Reply:
Yes those divs are just to give me a visual. They will be removed when the PR is finalised

Reply:
Sure. Just saying that styling up h1 globally could replace that div with that long class name

Reply:
Yep that can be done at a component level if we want as a h1 in one component may not be the same in another. We are trying to align the names with Abstract so we all know which styling we are talking about.

Reply:
My intuition says that we should use the HTML elements in a global level, and have a unique h1, h2, h3, p etc. across the application. And these will represent the same concepts different types of headers paragraphs on all pages. So we'll be able to reference them in a conversation/documentation.
What do you think?